website/static/js/logTextParser.js

```diff
@@ -242,26 +327,71 @@ var LogPieces = {
     },
     // The new title of node involved
     title_new: {
+        controller: function(logObject){
+            var self = this;
+            var nodeObject = logObject.embeds.original_node;
+
+            self.returnLinkForPath = function(){
+                if(paramIsReturned(nodeObject, logObject) && nodeObject.data){
+                    if (nodeObject.data.links && nodeObject.data.attributes) {
+                        return nodeObject.data.links.html;
+                    }
+                }
+                return null;
+            };
+        },
         view: function (ctrl, logObject) {
-            return returnTextParams('title_new', 'a title', logObject);
+            var url = ctrl.returnLinkForPath();
+            return returnTextParams('title_new', 'a title', logObject, url);
         }
     },
     // Update fields for the node
     updated_fields: {
         view: function (ctrl, logObject) {
-            return returnTextParams('updated_fields', 'field(s)', logObject);
-        }
-    },
+            var updatedFieldsParam = logObject.attributes.params.updated_fields;
+            if (paramIsReturned(updatedFieldsParam, logObject)) {
+                var updatedField = Object.keys(updatedFieldsParam)[0];
+                if (updatedField === 'category'){
+                    return m('span', updatedField + ' to ' + nodeCategories[updatedFieldsParam[updatedField].new]);
+                }
+                return m('span', updatedField + ' to ' + updatedFieldsParam[updatedField].new);
+                }
+            return m('span', 'field');
+        }},
     // external identifiers on node
     identifiers: {
         view: function (ctrl, logObject) {
-            return returnTextParams('identifiers', 'identifier(s)', logObject);
+            external_ids = logObject.attributes.params.identifiers;
+            if (paramIsReturned(external_ids, logObject)) {
+                var doi = external_ids.doi;
+                var ark = external_ids.ark;
+                if (doi && ark) {
+                    return m('span', 'doi:' + doi + ' and ark:' + ark);
```

Comment:
Is there any use on returning just one or the other? Or is this behavior defined by the project log templates?